File: Practica 1/exploracion.py
import os

os.chdir("/Users/lucky/Desktop/MAD/Practica 1")
print("Directorio actual:", os.getcwd())
print("Archivos en carpeta:", os.listdir())

import kagglehub
import pandas as pd

#Cargar dataset NKE
df = pd.read_csv("NKE.csv")

#Exploracion Inicial

#Ver dimensiones del datasheet
print("dimensiones: ")
df.shape

#Informacion general del datasheet
print("Informacion del dataset: ")
df.info()                              

#Ver las primeras filas
print("Primeras 10 filas del datasheet: ")
df.head(10)

#Estadisticas Descriptivas
print("Estadisticas descriptivas: ")
df.describe()

File: Practica 1/exploracion.r
# Cambiar directorio
setwd("/Users/lucky/Desktop/MAD/Practica 1")

# Ver directorio actual
cat("Directorio actual:", getwd(), "\n")

# Ver archivos en la carpeta
cat("Archivos en carpeta:\n")
print(list.files())

# Cargar dataset
df <- read.csv("NKE.csv")

# Exploración inicial

# Dimensiones
cat("Dimensiones:\n")
print(dim(df))

# Información general
cat("Información del dataset:\n")
print(str(df))

# Primeras filas
cat("Primeras 10 filas:\n")
print(head(df, 10))

# Estadísticas descriptivas
cat("Estadísticas descriptivas:\n")
print(summary(df))